notebooks/231-instruct-pix2pix-image-editing/231-instruct-pix2pix-image-editing.ipynb

```diff
@@ -0,0 +1,1062 @@
+{
```

Comment:
<p>I got error on this step. Is it need special version of transformers ?:</p><p><code>---------------------------------------------------------------------------</code></p><p><code>AttributeError &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Traceback (most recent call last)</code></p><p><code>&lt;ipython-input-6-1588a43b6b8f&gt; in &lt;module&gt;</code></p><p><code>&nbsp; &nbsp; &nbsp; 3 get_ipython().system('pip install ftfy &nbsp;')</code></p><p><code>&nbsp; &nbsp; &nbsp; 4 model_id = "timbrooks/instruct-pix2pix"</code></p><p><code>----&gt; 5 pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(model_id, torch_dtype=torch.float32, safety_checker=None)</code></p><p><code>&nbsp; &nbsp; &nbsp; 6 scheduler_config = pipe.scheduler.config</code></p><p><code>c:\users\sbalandi\me\work\openvino_env\lib\site-packages\diffusers\pipelines\pipeline_utils.py in from_pretrained(cls, pretrained_model_name_or_path, **kwargs)</code></p><p><code>&nbsp; &nbsp; 681 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; library = importlib.import_module(library_name)</code></p><p><code>&nbsp; &nbsp; 682 </code></p><p><code>--&gt; 683 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; class_obj = getattr(library, class_name)</code></p><p><code>&nbsp; &nbsp; 684 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; importable_classes = LOADABLE_CLASSES[library_name]</code></p><p><code>&nbsp; &nbsp; 685 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; class_candidates = {c: getattr(library, c, None) for c in importable_classes.keys()}</code></p><p><code>c:\users\sbalandi\me\work\openvino_env\lib\site-packages\transformers\utils\import_utils.py in __getattr__(self, name)</code></p><p><code>&nbsp; &nbsp;1067 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; value = getattr(module, name)</code></p><p><code>&nbsp; &nbsp;1068 &nbsp; &nbsp; &nbsp; &nbsp; else:</code></p><p><code>-&gt; 1069 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; raise AttributeError(f"module {self.__name__} has no attribute {name}")</code></p><p><code>&nbsp; &nbsp;1070 </code></p><p><code>&nbsp; &nbsp;1071 &nbsp; &nbsp; &nbsp; &nbsp; setattr(self, name, value)</code></p><p><code>AttributeError: module transformers has no attribute CLIPImageProcessor</code></p><p> </p><p>Also I got this, may be add it to the requirement ? :</p><p><code>Cannot initialize model with low cpu memory usage because <code>accelerate</code> was not found in the environment. Defaulting to <code>low_cpu_mem_usage=False</code>. It is strongly recommended to install <code>accelerate</code> for faster and less memory-intense model loading. You can do so with: </code></p><p><code><code></code><code></code></p><p><code>pip install accelerate</code></p><p><code></code><code></code></code></p><p><code>.</code></p><p><code>ftfy or spacy is not installed using BERT BasicTokenizer instead of ftfy.</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/796/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I had 4.24.0, it work fine with 4.26.0

Reply:
<p>looks like minimal transformers version is 4.25.1, thank you for note, I'll update requirements for this notebook</p>

Reply:
Updated